charts/retool/charts/retool-temporal-services-helm/templates/server-deployment.yaml

```diff
@@ -184,6 +184,9 @@ spec:
             - name: TEMPORAL_VERSION_CHECK_DISABLED
               value: "1"
           {{- end }}
+          {{- with .Values.environmentVariables }}
+{{ toYaml . | indent 10 }}
```

Comment:
10 or 12? 

can template this to make sure with `helm template -f ~/retool-helm/charts/retool/values.yaml --set config.encryptionKey="foo"  foo ~/retool-helm/charts/retool` (replacing with appropriate chart and paths etc.)

Reply:
yup hit this when testing; great catch!